src/app/modules/item/components/item-children-edit/item-children-edit.component.ts

```diff
@@ -109,6 +109,7 @@ export class ItemChildrenEditComponent implements OnInit, OnDestroy, OnChanges {
       })
     ),
     mapToFetchState({ resetter: this.refresh$ }),
+    share(),
```

Comment:
What's the purpose of this `share()` ? You encountered xhr requests duplicates ?

Reply:
Yes